core/packages/test/config/beefy-relay.json

```diff
@@ -8,8 +8,7 @@
   },
   "sink": {
     "ethereum": {
-      "endpoint": "ws://127.0.0.1:8546",
-      "gas-limit": 5000000
```

Comment:
would like to know is this required for local setup, remove it since does not work for goerli

Reply:
from tests seems still required for local setup, so set as variable here